src/tiler/server.js

```diff
@@ -24,7 +24,7 @@ var config = {
         }
     },
     renderCache: {
-        ttl: 60000, // seconds
```

Comment:
You are right.  @lliss and I were testing yesterday, and that did not get changed back.